Allows this test to pass when twisted is installed.  $TRIAL_PYTHONPATH
must be existing writable dir and in PYTHONPATH

$OpenBSD: patch-twisted_trial_test_test_loader_py,v 1.1 2011/06/20 10:25:32 eric Exp $
--- twisted/trial/test/test_loader.py.orig	Wed May  4 14:48:38 2011
+++ twisted/trial/test/test_loader.py	Wed May  4 14:50:10 2011
@@ -166,7 +166,7 @@ class FileTest(packages.SysPathManglingTest):
         Test loader against a filesystem directory. It should handle
         'path' and 'path/' the same way.
         """
-        path  = util.sibpath(__file__, 'goodDirectory')
+        path  = os.path.join(os.getenv('TRIAL_PYTHONPATH'), 'goodDirectory')
         os.mkdir(path)
         f = file(os.path.join(path, '__init__.py'), "w")
         f.close()
